src/main/java/org/jabref/logic/ai/templates/AiTemplate.java

```diff
@@ -0,0 +1,30 @@
+package org.jabref.logic.ai.templates;
+
+import org.jabref.logic.l10n.Localization;
+
+public enum AiTemplate {
+    // System message that is applied in the AI chat.
+    CHATTING_SYSTEM_MESSAGE,
+
+    // Template of a last user message in the AI chat with embeddings.
+    CHATTING_USER_MESSAGE,
+
+    // Template that is used to summarize the chunks of text.
+    SUMMARIZATION_CHUNK,
+
+    // Template that is used to combine the summarized chunks of text.
+    SUMMARIZATION_COMBINE;
+
+    public String getLocalizedName() {
+        return switch (this) {
+            case CHATTING_SYSTEM_MESSAGE ->
+                    Localization.lang("System message for chatting");
+            case CHATTING_USER_MESSAGE ->
+                    Localization.lang("User message for chatting");
+            case SUMMARIZATION_CHUNK ->
+                    Localization.lang("Summarization chunk");
+            case SUMMARIZATION_COMBINE ->
+                    Localization.lang("Summarization combine");
+        };
```

Comment:
We can actually define fields in enums, so here we can define a field called localizedName and define the relevant constructor. After that we can write something like this: `CHATTING_SYSTEM_MESSAGE(Localization.lang("System message for chatting"))`

Reply:
But I'm not sure, if the names are properties of templates, and especially localized ones

Reply:
If that is the case we can do something else, but if all strings are static having it as a field is better IMO.

Reply:
Just call `Localization.lang` in the enum. Why not? -- The initialization of the enum should take care about the localiaztion. When using `toString`, the stored localized string is returned.

I know this will cause troubles when users change the UI language. Since language switch is not done that often, I accept this.

---

I think, a related discussion was going on at https://github.com/JabRef/jabref/pull/8304 and [ADR-0023](https://github.com/JabRef/jabref/blob/unlocalized-preferences-new/docs/adr/0023-unlocalized-preferences.md).